test/h3core.spec.js

```diff
@@ -991,65 +1001,64 @@ test('h3GetFaces', assert => {
     assert.end();
 });
 
-test('h3GetBaseCell', assert => {
+test('getBaseCellNumber', assert => {
     const h3Index = '8928308280fffff';
 
-    assert.equal(h3.h3GetBaseCell(h3Index), 20, 'Got expected base cell');
+    assert.equal(h3.getBaseCellNumber(h3Index), 20, 'Got expected base cell');
 
     assert.end();
 });
 
-test('h3ToParent', assert => {
+test('cellToParent', assert => {
     // NB: This test will not work with every hexagon, it has to be a location
     // that does not fall in the margin of error between the 7 children and
     // the parent's true boundaries at every resolution
     const lat = 37.81331899988944;
     const lng = -122.409290778685;
     for (let res = 1; res < 10; res++) {
         for (let step = 0; step < res; step++) {
-            const child = h3.geoToH3(lat, lng, res);
-            const comparisonParent = h3.geoToH3(lat, lng, res - step);
-            const parent = h3.h3ToParent(child, res - step);
+            const child = h3.latLngToCell(lat, lng, res);
+            const comparisonParent = h3.latLngToCell(lat, lng, res - step);
+            const parent = h3.cellToParent(child, res - step);
 
             assert.equals(parent, comparisonParent, `Got expected parent for ${res}:${res - step}`);
         }
     }
     assert.end();
 });
 
-test('h3ToParent - Invalid', assert => {
+test('cellToParent - Invalid', assert => {
     const h3Index = '8928308280fffff';
 
-    assert.equals(h3.h3ToParent(h3Index, 10), null, 'Finer resolution returns null');
-    assert.equals(h3.h3ToParent(h3Index, -1), null, 'Invalid resolution returns null');
-    assert.equals(h3.h3ToParent('foo', 10), null, 'Invalid index returns null');
-
+    assert.throws(() => h3.cellToParent(h3Index, 10), /\(12\)$/, 'Throws on finer resolution');
+    assert.throws(() => h3.cellToParent(h3Index, -1), /\(4\)$/, 'Throws on invalid resolution');
+    assert.throws(() => h3.cellToParent('foo', 10), /\(12\)$/, 'Throws on invalid index');
     assert.end();
 });
 
-test('h3ToChildren', assert => {
+test('cellToChildren', assert => {
```

Comment:
Looks like we're missing error-handling tests for this

Reply:
There are some on L1047 and L1048 but there can always be more tests